<launch>
    <arg name="launch_state" default="sim"/>

    <node pkg="load_waypoints" type="nav_options.py" name="nav_control" output="screen" />

</launch> 
